e2e-patient-test/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
```

Comment:
Larger question does this need to be a separate JAR from the other e2e tests?

Reply:
Trying to use the existing e2e tests and have the correct configuration for the SpringBootTest that the patient endpoint tests needs was challenging and I didn't want to mess too much with the e2e tests.